css/styles.css

```diff
@@ -0,0 +1,65 @@
+
+body {background-color:#FFFFFF; 
```

Comment:
use css standard coding conventions and best practices to organize code.
http://www.phpied.com/css-coding-conventions/